src/app/grid/grid-select/grid-select-sample.component.scss

```diff
@@ -0,0 +1,4 @@
+:host {
+    display: block;
+    padding: 16px;
+}
```

Comment:
I expect this PR should include the igxSelect styling ( last committed in the master branch ) via https://github.com/IgniteUI/igniteui-angular-samples/pull/2440/commits/3e8545121551d184b912f1916eac14e16641c001